Add issue template (#31)
fastruby/styleguide: .github/ISSUE_TEMPLATE/bug_report.md

---
name: Bug report
about: Create a report to help us improve
title: "[BUG] "
labels: 'bug'
assignees: 'guardians-of-the-galaxy'

---

**IMPORTANT: please make sure you ask yourself all intro questions and fill all sections of the template.**

**Before we start...:**

- [ ] I checked the documentation and found no answer
- [ ] I checked to make sure that this issue has not already been filed
- [ ] I'm reporting the issue to the correct repository (for multi-repository projects)

**Version, Branch, or Commit:**

> Inform what version, branch, commit of Skunk you are using.

Reviewer: Skunk?

**Expected behavior:**

Please include a detailed description of the behavior you were expecting when you encountered this issue.

**Actual behavior:**

Please include a detailed description of the actual behavior of the application.

**Steps to reproduce:**

> How do I achieve this behavior? Use the following format to provide a step-by-step guide:

Reviewer: ```suggestion
How did you achieve this behavior? Use the following format to provide a step-by-step guide:
```
